requirements/system/systemmodel/system-model.tex

```diff
@@ -0,0 +1,26 @@
+\subsection{System Model}
+\begin{figure}
+  \subsubsection{Schedule new Job}
+  \centering
+  \includegraphics[width=\linewidth]{./system/systemmodel/images/schedulejob.png}
+  \caption{This diagram depicts how a user schedules a job, which the CS distributes to WC 1.
+  In that case CS knows that there are enough resurces on WC 1 to execute users job.
+  At the end the CS notifies the user that the job has finished.}
+\end{figure}
+
+\begin{figure}
+  \subsubsection{Access a running job}
+  \centering
+  \includegraphics[width=\linewidth]{./system/systemmodel/images/accessjob.png}
+  \caption{This diagram depicts how a user accesses his running job.
+  After starting the job, the CS returns WC ip and container id.
+  Using this information the user can connect to the container executing his job at any time.
+  Only the user that started the job and administrators can access the job.}
```

Comment:
How does the user authentification on the work machine work?

Reply:
I'm not sure how that is going to work. Probably we can give the user a password to the container when he starts it, so he can connect to it later, but then what happens with the administrators.